tests/BenchmarkDotNet.IntegrationTests/AllSetupAndCleanupTest.cs

```diff
@@ -62,6 +63,42 @@ public void AllSetupAndCleanupMethodRunsTest()
             Assert.Equal(expectedLogLines, actualLogLines);
         }
 
+        [Fact]
+        public void AllSetupAndCleanupMethodRunsAsyncTest()
+        {
+            var logger = new OutputLogger(Output);
+            var miniJob = Job.Default.With(RunStrategy.Monitoring).WithWarmupCount(2).WithIterationCount(3).WithInvocationCount(1).WithUnrollFactor(1).WithId("MiniJob");
+            var config = CreateSimpleConfig(logger, miniJob);
+
+            CanExecute<AllSetupAndCleanupAttributeBenchmarksAsync>(config);
+            Output.WriteLine(OutputDelimeter);
```

Comment:
I understand that re-writing this to `Output` is for debugging purposes?

Should we keep it in the code?

Reply:
I just copy pasted the previous test and only changed the class that was used. No idea what that was added but we can remove it without any issues.